imod/mf6/auxiliary_variables.py

```diff
@@ -25,11 +25,40 @@ def _get_auxiliary_data_variable_names_mapping(package):
     return result
 
 
-def add_periodic_auxiliary_variable(package):
+def add_periodic_auxiliary_variable(package) -> None:
+    """
+    splits an auxiliary dataarray (with one or more auxiliary variable dimension) into dataarrays per
+    auxiliary variable dimension. For example a concentration auxiliary variable in a flow package
+    will have a species dimension, and will be split in several dataarrays- one for each species.
+    """
     if hasattr(package, "_auxiliary_data"):
         for aux_var_name, aux_var_dimensions in package._auxiliary_data.items():
             aux_coords = package.dataset[aux_var_name].coords[aux_var_dimensions].values
             for s in aux_coords:
                 package.dataset[s] = package.dataset[aux_var_name].sel(
                     {aux_var_dimensions: s}
                 )
+
+
+def remove_periodic_auxiliary_variable(package) -> None:
+    """
+    removes the data arrays created by add_periodic_auxiliary_variable(...) but does not
+    remove the auxiliary dataarray used as source for add_periodic_auxiliary_variable(...)
```

Comment:
Use the following form when refering to another method to automatically generate a link in sphinx:
`:meth:`imod.mf6.auxiliary_variables.add_periodic_auxiliary_variable``

Reply:
done